### PR TITLE
Upgrade node 5.12.0

### PR DESCRIPTION
**Pull request by jackmarchant**
@expert360/dev please review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ Docker container for a Node and Ruby development workflow
  * [`0.10_1.9` (0.10_1.9/Dockerfile)](https://github.com/expert360/docker-node-ruby/blob/master/0.10_1.9/Dockerfile)
  * [`0.12_2.2` (0.12_2.2/Dockerfile)](https://github.com/expert360/docker-node-ruby/blob/master/0.12_2.2/Dockerfile)
  * [`4.4.7_2.2` (4.4.7_2.2/Dockerfile)](https://github.com/expert360/docker-node-ruby/blob/master/4.4.7_2.2/Dockerfile)
+ * [`5.5.0_2.2` (4.4.7_2.2/Dockerfile)](https://github.com/expert360/docker-node-ruby/blob/master/4.4.7_2.2/Dockerfile)
```
Comment (seguer) [inline README.md]: Wrong link

File: 5.5.0_2.2/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,82 @@
+FROM buildpack-deps:jessie
+
+MAINTAINER Jack <jack@expert360.com>
+
+# ====================
+# Install Nodejs 5.5.0 LTS
```
Comment (seguer) [inline 5.5.0_2.2/Dockerfile]: Where'd you find the Dockerfile for 5.5.0? I can't find it in the official repo.

Comment (jackmarchant) [inline 5.5.0_2.2/Dockerfile]: judging from this link https://hub.docker.com/_/node/ it doesn't look like there is one for that version. Could we go higher even if they're out of sync?

**Comment by jackmarchant** [comment]
@seguer updated

**Comment by seguer** [comment]
LGTM

**Pull request merged**